style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
The CSS order rules are annoying but once you learn them they're not too hard to work around. It's just specificity, ties broken by bottom-first.